packages/gatsby-source-drupal/src/utils.js

```diff
@@ -337,5 +343,37 @@ ${JSON.stringify(nodeToUpdate, null, 4)}
   }
 }
 
+let hasLogs = false
```

Comment:
```suggestion
let hasLoggedContentSyncWarning = false
```

Reply:
resolved in https://github.com/gatsbyjs/gatsby/pull/33683/commits/a02237f9fa26af5fa58f4ab23eff2fd51802e3aa